src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceFacadeImpl.java

```diff
@@ -199,52 +196,56 @@ private CaseWorkerFileCreationResponse createResponse(int noOfUploadedRecords,
                             .errorDescription(jsrInvalid.getErrorDescription())
                             .filedInError(jsrInvalid.getFieldInError()).build())));
 
-            noOfUploadedRecords = noOfUploadedRecords - failedRecords.size();
-            String suspendedRecordMessage = getSuspendedErrorMessageForJsr(isCaseWorker, failedRecords);
-
+            String detailedMessage = constructDetailedMessage(totalRecords, suspendedRecords, failedRecords);
             return caseWorkerFileCreationResponseBuilder.message(REQUEST_FAILED_FILE_UPLOAD_JSR)
-                .detailedMessage(format(RECORDS_FAILED, failedRecords.size())
-                    + recordsUploadMessage(noOfUploadedRecords, true)
-                    + suspendedRecordMessage).errorDetails(jsrFileErrors).build();
+                .detailedMessage(detailedMessage).errorDetails(jsrFileErrors).build();
         } else {
-            String suspendedRecordMessage = getSuspendedErrorMessage(isCaseWorker,
-                suspendedRow, false);
+            StringJoiner detailedMessage = new StringJoiner(SPACE + AND + SPACE);
```

Comment:
is this not this string add start?

Reply:
No, it will add only if there are more than one string